src/binding.gyp.in

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
How close to json does this file need to be?  If this file works then disregard this comment, however you may need to run this by a json lint checker to clean it up a bit.